pkg/scheduler/algorithm/priorities/interpod_affinity_test.go

```diff
@@ -295,7 +295,7 @@ func TestInterPodAffinityPriority(t *testing.T) {
 				{ObjectMeta: metav1.ObjectMeta{Name: "machine3", Labels: labelAzAz1}},
 			},
 			expectedList: []schedulerapi.HostPriority{{Host: "machine1", Score: schedulerapi.MaxPriority}, {Host: "machine2", Score: 0}, {Host: "machine3", Score: 0}},
-			test: "Affinity: pod that matches topology key & pods in nodes will get high score comparing to others" +
+			name: "Affinity: pod that matches topology key & pods in nodes will get high score comparing to others" +
```

Comment:
I know you tried not to modify existing test names, but I think "Affinity:" is redundant in these test names. If you like, we can do test name cleanup in a separate PR (after this one)